virttest/qemu_devices/qcontainer.py

```diff
@@ -930,6 +930,10 @@ def get_aavmf_vars(params):
             devices.append(qdevices.QStringDevice('machine', cmdline=cmd,
                                                   child_bus=bus,
                                                   aobject="virtio-mmio-bus"))
+            vga = params.get('vga')
+            if vga and vga != 'none':
+                # Always add mouse when 'vga' is set
+                devices.append(qdevices.QDevice("virtio-mouse-device"))
```

Comment:
I'm not sure whether we need to check the device is supported before attaching to the vm, do you think it is necessary?

Reply:
Not sure either. I'd assume someone who chooses `vga=virtio` to have fairly recent qemu. Otherwise they should disable `vga` completely.